react/week3/todoform/src/Modal/TodoList.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react'
+import Todos from './Todos';
+import axios from 'axios'
+
+function TodoList() {
+    const [todos, setTodos] = useState([]);
+    
+    useEffect(()=> {
+        axios.get('https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw')
```

Comment:
why you are not using async to call API as this may impact performance issue.

Reply:
Great, I have changed Axios n used aync await function,.,.in the reviewed PR,.Thanks